docs/semgrep-assistant/overview.md

```diff
@@ -114,7 +114,6 @@ For more details, see the [Semgrep Assistant FAQ](https://get.semgrep.dev/assist
 
 Semgrep Assistant prompts you for feedback whenever it suggests that a finding is a false positive. Because Assistant content is generated by language models (LLMs), your feedback helps the Semgrep team improve Assistant.
 
-* In Semgrep Cloud Platform, the Assistant recommendation appears under **Activity** for a finding, along with **Agree and ignore** or **Disagree** buttons.
```

Comment:
I haven't been able to confirm this happens any longer. If we can find confirmation, we can restore this line.

Reply:
I don't think we intentionally removed this but I'm not seeing it now either. Fine to remove for now to unblock this.

Reply:
Confirmed it's a bug, hopefully won't take too long to fix, and then we can merge with the line restored.